src/root/port.d

```diff
@@ -21,11 +21,7 @@ extern (C) real strtold(const(char)* p, char** endp);
 
 extern (C++) struct Port
 {
-    enum nan = double.nan;
-    enum infinity = double.infinity;
```

Comment:
IIRC I wrote the D version of port by adding members as I saw errors, rather than anything more systematic, so a lot of the code is quite old (in terms of ddmd) and doesn't really attempt to match the C++ source apart from name and value.